### PR TITLE
chore: update read me and launch.json

### PR DESCRIPTION
**Pull request by utsabc**
## Description of the change

Updating ReadMe.md to add the steps for building transformer with typescript changes, fix launch.json for debugging transformer

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,7 +22,8 @@ On Linux, install the required dependencies `python`, `make` and `g++` and follo
 
 1. Clone this repository
 2. Run `npm run setup`
-3. Start the server with `npm start`
+3. Build by running `npm run build:clean`
```
Comment (saikumarrs) [inline README.md:25]:
```suggestion
3. Build the service with `npm run build:clean`
```

File: .vscode/launch.json [hunk 2]
```diff
@@ -19,7 +19,6 @@
       "name": "Launch TS",
       "program": "${workspaceFolder}/dist/src/index.js",
       "outFiles": ["${workspaceFolder}/dist/src/**/*.js"],
-      "preLaunchTask": "npm run build",
```
Comment (sanpj2292) [inline .vscode/launch.json:22]: [non-blocking] Why did we remove this ?

**Review by sanpj2292 — DISMISSED** [review]

File: README.md [hunk 3]
```diff
@@ -23,7 +22,8 @@ On Linux, install the required dependencies `python`, `make` and `g++` and follo
 
 1. Clone this repository
 2. Run `npm run setup`
```
Comment (saikumarrs) [inline README.md:24]:
```suggestion
2. Setup the repository with `npm run setup`
```

File: README.md [hunk 4]
```diff
@@ -44,7 +44,8 @@ On Linux, install the required dependencies `python`, `make` and `g++` and follo
...
-3. Start the server with `ENABLE_FUNCTIONS=false npm start`
+3. Build by running `npm run build:clean`
+4. Start the server with `ENABLE_FUNCTIONS=false npm start`
```
Comment (saikumarrs) [inline README.md:48]: Same as above.

**Review by saikumarrs — APPROVED** [review]

**Pull request merged**